typescript/infra/config/environments/mainnet/helloworld/abacus/addresses.json

```diff
@@ -1,26 +0,0 @@
-{
```

Comment:
why delete all these types of files? bc we'll need to redeploy them for v2?

Reply:
Yes exactly